### PR TITLE
Add Liquibase

### PR DESCRIPTION
**Pull request by SemyonKorostik**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -35,6 +41,15 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+
+			<plugin>
+				<groupId>org.liquibase</groupId>
+				<artifactId>liquibase-maven-plugin</artifactId>
+				<version>4.4.0</version>
```
Comment (AntonMaksimenkau) [inline pom.xml]: Let's put the liquibase version in the properties section.
https://maven.apache.org/pom.html#Properties
Plugin and dependency versions are highly coupled, so it makes sense to store it at one place

Comment (AntonMaksimenkau) [reply pom.xml]: Let's put the first liquibase script which will create the database.
You can find the sql here: https://github.com/mtalstykh-exadel/team4_docs/blob/main/architecture/backend/database-sceme/db/language-testing.sql

File: src/main/resources/db/changelog/db.changelog-master.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mpasenko) [inline src/main/resources/db/changelog/db.changelog-master.xml]: Please rename file. Good practice to is when file hase name related to ticket. For instance: if you are going to create database schema your file will have name 'create-db.xml'

File: src/main/resources/db/changelog/db.changelog-master.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.4.xsd">
+    
+    <preConditions>
```
Comment (mpasenko) [inline src/main/resources/db/changelog/db.changelog-master.xml]: I think this preCondition is redundant. You can define DBMS in properties file

File: pom.xml [hunk 4]
```diff
@@ -1,41 +1,57 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (AntonMaksimenkau) [inline pom.xml:3]: please, use a common code style

**Review by AntonMaksimenkau — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties [hunk 5]
```diff
@@ -1 +1,9 @@
+#Liquibase
+spring.liquibase.change-log=classpath:db/changelog/db-changelog.xml
+logging.level.liquibase=DEBUG
```
Comment (mpasenko) [inline src/main/resources/application.properties]: remove pls

**Review by mpasenko — DISMISSED** [review]

**Review by Darina007 — DISMISSED** [review]

**Review by AntonMaksimenkau — APPROVED** [review]

**Review by AntonMaksimenkau — APPROVED** [review]

**Review by Darina007 — APPROVED** [review]

**Review by maxim12321 — APPROVED** [review]

**Pull request merged**